zc_install/includes/classes/class.zcDatabaseInstaller.php

```diff
@@ -352,20 +352,31 @@ public function parserRenameTable()
   {
     if (!$this->tableExists($this->lineSplit[2]))
     {
-      if (!isset($result)) $result = sprintf(REASON_TABLE_NOT_FOUND, $table).' CHECK PREFIXES!';
+      if (!isset($result)) $result = sprintf(REASON_TABLE_NOT_FOUND, $this->lineSplit[2]).' CHECK PREFIXES!';
       $this->writeUpgradeExceptions($this->line, $result, $this->fileName);
       $this->ignoreLine = true;
     } else {
       if ($this->tableExists($this->lineSplit[4]))
       {
-        if (!isset($result)) $result = sprintf(REASON_TABLE_ALREADY_EXISTS, $table);
+        if (!isset($result)) $result = sprintf(REASON_TABLE_ALREADY_EXISTS, $this->lineSplit[4]);
         $this->writeUpgradeExceptions($this->line, $result, $this->fileName);
         $this->ignoreLine = true;
       } else {
         $this->line = 'RENAME TABLE ' . $this->dbPrefix . $this->lineSplit[2] . ' TO ' . $this->dbPrefix . substr($this->line, (13 + strlen($this->lineSplit[2]) + 4));
       }
     }
   }
+  public function parserLeftJoin()
+  {
+    if (!$this->tableExists($this->lineSplit[2]))
+    {
+      if (!isset($result)) $result = sprintf(REASON_TABLE_NOT_FOUND, $this->lineSplit[2]).' CHECK PREFIXES!';
+      $this->writeUpgradeExceptions($this->line, $result, $this->fileName);
+      error_log($result . "\n" . $this->line . "\n---------------\n\n");
```

Comment:
Did you mean to leave this `error_log` call?

Reply:
Now able to respond in this location, I had intended the error_log to be here so that a `zcInstallDEBUG-xxxxxxxxxx-xxxxxx` file would be created in addition to populating a `zcInstallLog_xx-xx-xxxx_xx-xx-xx-xxxxxx` file though both are populated with information in this situation.